packages/calcite-components/src/components/tab-nav/assets/tab-nav/t9n/messages.json

```diff
@@ -0,0 +1,4 @@
+{
+  "nextTabTitles": "Next tab titles",
```

Comment:
+@geospatialem @SkyeSeitz @ashetland for editorial review. We can make edits in a follow-up PR for October.